apps/website/inertia/components/home/highlights/highlight_card.tsx

```diff
@@ -0,0 +1,64 @@
+import { Badge } from "~/components/ui/badge";
+import { EventType } from "./highlights";
+import { cva } from "class-variance-authority";
+import { cn } from "~/lib/utils";
+
+interface HighlightCardProps {
+  title: string;
+  person: string;
+  schedule: string;
+  type: EventType;
+  image: string;
+}
+
+const highlightCardVariants = cva(
+  "border border-2 rounded-xl w-fit py-0 max-w-xl h-fit h-[16.35em]",
```

Comment:
On mobile the card components become wider than the screen, which looks, in my opinion, kinda clunky.

On a Galaxy S20 Ultra Android 11: 
![image](https://github.com/user-attachments/assets/82d3816a-35dc-4b09-8fa8-52e7021258dd)

On an iPhone 12/13 Pro Max iOS 14.6: 
![image](https://github.com/user-attachments/assets/3eb15ac3-7b55-450b-955b-78b4990dab9e)

The immediate suggestion would be to add word-wrapping to the text or possibly constrain the `max-width` of these elements on mobile displays.

Using this change
```suggestion
  "border border-2 rounded-xl w-fit py-0 max-w-sm md:max-w-xl h-fit h-[16.35em]",
```
yields 
![image](https://github.com/user-attachments/assets/d636ed42-6a38-434e-b323-24d1b78f3eb3)
which looks neater in my opinion (minus the obvious overflowing text on the badge).

Reply:
The figma file shows the card in mobile like I did it but your change also looks good.

Reply:
one possible solution for this sort of issue would be to make the image the size shifting component rather than the text, seeing as personally seeing a 3-line text-wrap doesn't look very appealing (keeping it to 2 lines would be ideal but obviously larger titles require more wrapping). regardless, as time is of the essence the current solution would be suitable if no other can be achieved in time (with the fix ((probably a `w-fit` on that element would be enough)) of the "palestra" bubble :P).

Reply:
I've now changed it to your suggestion @Naapperas and the `w-fit` on the "palestra" bubble is already there

Reply:
I didn't use a carousell, but I should have... I can try to change it and make autoplay

Reply:
But I dont know how easy it will be

Reply:
would be a nice to have, if there are no other pressing matters. but as always I leave it to your discretion.

Reply:
Two ways of achieving this when using shadcn's carousel:
- use an effect and every X seconds call `https://www.embla-carousel.com/api/methods/#scrollnext`
- OR install the autoplay plugin (https://www.embla-carousel.com/api/plugins/)
@tomaspalma

Reply:
https://github.com/user-attachments/assets/3f416bbb-2f5a-434b-871a-41050262145c

I did it like this. Please tell me if you like it

Reply:
I like it